packages/typespec-client-generator-core/design-docs/flags.md

```diff
@@ -0,0 +1,35 @@
+# TCGC `tspconfig.yaml` flags
```

Comment:
I'd like to clarify the context of tcgc flags here, does that mean 1) tcgc would use these flags or 2) the common flags cross all emitters?

Reply:
the flag tcgc will use and get from emitter's config in `tspconfig.yaml`.